crates/arrow_tools/src/schema.rs

```diff
@@ -72,3 +73,21 @@ pub fn verify_schema(
 
     Ok(())
 }
+
+#[must_use]
+pub fn transform_schema(schema: &Schema) -> Schema {
```

Comment:
```suggestion
pub fn expand_views_schema(schema: &Schema) -> Schema {
```

Maybe a better name?